lambda/extension/client/client.go

```diff
@@ -110,30 +116,31 @@ func (ic *InvocationClient) getNextEventURL() string {
 	return fmt.Sprintf("http://%s/%s/extension/event/next", ic.baseUrl, ic.version)
 }
 
-// getInitErrorURL returns the Lambda Extension next event URL
+// getInitErrorURL returns the Lambda Extension initialization error URL
 func (ic *InvocationClient) getInitErrorURL() string {
 	return fmt.Sprintf("http://%s/%s/extension/init/error", ic.baseUrl, ic.version)
 }
 
-// getExitErrorURL returns the Lambda Extension next event URL
+// getExitErrorURL returns the Lambda exit error URL
 func (ic *InvocationClient) getExitErrorURL() string {
 	return fmt.Sprintf("http://%s/%s/extension/exit/error", ic.baseUrl, ic.version)
```

Comment:
Does it matter if this simulated string is http or https?

Reply:
This is a part of the Lambda spec. The endpoint is local to the Lambda extension.